experimental/canvas/explorations-py/courses-extractor/README.md

```diff
@@ -0,0 +1,32 @@
+# Python-Base API Exploration
+
+Be sure to have Python 3 and poetry installed.
+
+## Environment Setup
+
+1. In the explorations-py folder run the command:
+
+    ```python
+    poetry install
+    ```
+
+1. In vscode type `ctrl`+`shift`+`p` and search for the command `Python: select interpreter`, then point it to your generated folder `explorations-py/.venv`.
```

Comment:
Is this instruction really necessary, since we have the python path already spelled out in the settings.json file?

Reply:
I found it useful when I have the entire repository opened in vscode, so I can switch between projects without needing to open a new VSCode instance.

Reply:
ok